File: HRA.Web/ClientApp/src/app/models/hraDetails.ts
import { Questions } from './questions';

export class HRADetails {
  groupId: string;
  firstName: string;
  lastName: string;
  middleInitial: string;
  date: Date;
  dob: Date;
  ssn: string;
  language: string;
  department: string;
  position: string;
  gender: string;
  email: string;
  alternateEmail: string;
  apartmentNumber: string;
  mailingAddress: string;
  city: string;
  state: string;
  zipCode: string;
  country: string;
  currentDiagnosis: Questions[];
  otherDiagnosis: string;
  textMessageConfirmation: boolean;
  contactMethodPreference: string;
  cellPhone: string;
  alternateCellPhone: string;
  bestTimeToCall: string;
  heightInInches: 0;
  heightInFeet: 0;
  weightInLbs: 0;
  bmi: 0;
  mostRecentCheckupDate: Date;
  healthQuestions: Questions[];
  comments: string;
}

File: HRA.Common/Models/HRADetails.cs
using HRA.Common.Enums;
using System;

namespace HRA.Common.Models
{
    public class HRADetails : BaseModel
    {
        public string FirstName { get; set; }

        public string MiddleName { get; set; }

        public string LastName { get; set; }

        public DateTime Dob { get; set; }

        public string Ssn { get; set; }

        public string Email { get; set; }

        public string PhoneNumber { get; set; }

        public Gender Gender { get; set; }

        public string ExtraInfo { get; set; }

        public int GroupId { get; set; }
    }
}
